code/modules/tgui_panel/external.dm

```diff
@@ -35,3 +35,12 @@
 	// Force show the panel to see if there are any errors
 	winset(src, "output", "is-disabled=1&is-visible=0")
 	winset(src, "browseroutput", "is-disabled=0;is-visible=1")
+
+/client/verb/refresh_tgui()
+	set name = "Refresh TGUI"
+	set category = "OOC"
+
+	message_admins("start")
```

Comment:
epic

Reply:
Local admins spammed with this one simple trick